sockeye/loss.py

```diff
@@ -68,24 +68,24 @@ class CrossEntropyLoss(Loss):
     def __init__(self, normalize: bool = False):
         self._normalize = normalize
 
-    def get_loss(self, logits: mx.sym.Symbol, labels: mx.sym.Symbol) -> mx.sym.Symbol:
+    def get_loss(self, logits: mx.sym.Symbol, labels: mx.sym.Symbol) -> List[mx.sym.Symbol]:
         """
         Returns loss and softmax output symbols given logits and integer-coded labels.
 
         :param logits: Shape: (batch_size * target_seq_len, target_vocab_size).
         :param labels: Shape: (batch_size * target_seq_len,).
-        :return: Loss and softmax output symbols.
+        :return: List of loss and softmax output symbols.
         """
         if self._normalize:
             normalization = "valid"
         else:
             normalization = "null"
-        return mx.sym.SoftmaxOutput(data=logits,
+        return [mx.sym.SoftmaxOutput(data=logits,
```

Comment:
I cannot find any clear documentation on this. What does SoftmaxOutput return exactly? If it's a tuple, then here we are creating a list with a single (tuple) element.